src/fmllauncher/java/net/minecraftforge/fml/loading/moddiscovery/ModDiscoverer.java

```diff
@@ -86,8 +86,4 @@ public BackgroundScanHandler discoverMods() {
         loadingModList.setBrokenFiles(brokenFiles);
         return backgroundScanHandler;
     }
-
-    public void addExplodedTarget(final Path compiledClasses, final Path forgemodstoml) {
```

Comment:
?

Reply:
Whoops, didn't mean to put this in this PR. It's just an unused method